rotkehlchen/chain/avalanche/manager.py

```diff
@@ -125,7 +126,11 @@ def get_transaction_receipt(
             tx_receipt['status'] = 1 if successful else 0
             tx_receipt['transactionIndex'] = 0
             txhash = tx_receipt.pop('tx_hash')
-            tx_receipt['hash'] = txhash
+            tx_receipt['hash'] = deserialize_evm_tx_hash(txhash)  # covalent returns string
+            tx_receipt['from'] = to_checksum_address(tx_receipt['from_address'])
+            tx_receipt['to'] = to_checksum_address(tx_receipt['to_address'])
+            tx_receipt['gasPrice'] = tx_receipt['gas_price']
+            tx_receipt['gas'] = tx_receipt['gas_offered']
```

Comment:
at line 143 shouldn;t also `KeyError` be caught and appropriateley handled?

Keep in mind key error needs a bit of a different error message as you can see from other places in the code where we handle it since `str(e)` just shows the key that was not found

Reply:
Makes sense